components/calendar/calendar-styles.js

```diff
@@ -0,0 +1,83 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+
+export const calendarStyles = css`
+	:host {
+		display: block;
+		min-width: 350px;
+	}
+
+	table {
+		border-collapse: collapse;
+		border-spacing: 0;
+		table-layout: fixed;
+		width: 100%;
+	}
+
+	th[role="columnheader"] {
+		border-bottom: 1px solid var(--d2l-color-gypsum);
+		padding-bottom: 0.6rem;
+		padding-top: 0.3rem;
+	}
+
+	abbr {
+		text-decoration: none;
+	}
+
+	.d2l-calendar {
+		border-radius: 4px;
+	}
+
+	.d2l-calendar-title {
+		border-top-left-radius: 4px;
+		border-top-right-radius: 4px;
+		text-align: center;
+	}
+
+	.d2l-calendar-title .d2l-heading-4 {
+		height: 100%;
+		margin: 0;
+	}
+
+	.d2l-calendar-date {
+		align-items: center;
+		border: 2px solid transparent;
+		border-radius: 8px;
+		color: var(--d2l-color-ferrite);
+		cursor: pointer;
+		display: flex;
+		font-size: 0.8rem;
+		height: 2rem;
+		justify-content: center;
+		margin-left: auto;
+		margin-right: auto;
+		position: relative;
+		text-align: center;
+		width: 2rem;
+	}
+
+	.d2l-calendar-date:hover,
+	.d2l-calendar-date.d2l-calendar-date-hover {
+		background-color: var(--d2l-color-gypsum);
+	}
+
+	.d2l-calendar-date:focus {
+		border: 2px solid var(--d2l-color-celestine);
+		outline: none;
+	}
+
+	.d2l-calendar-date-first-row {
```

Comment:
I don't know that that would work given the way it's currently implemented since technically the first `tr` is the row with the days of the week (this one would more accurately be something like `d2l-calendar-date-first-date-row`)

Reply:
Hmm, what about something along the lines of `tbody > tr:first` though? I'm assuming the days of the week are in the `thead`.

Reply:
They are not currently in `thead` (I had just implemented the structure of what was in the monolith, which has them in `tbody`) but they definitely feel like they should be.

Reply:
Yeah seems like a worthwhile change.